### PR TITLE
Add initial Feb 2022 newsletter

### PR DESCRIPTION
**Pull request by ericholscher**
I've revamped the format a little bit to be less focused on each of our own goals,
and to simplify writing it each month.

### CODE REVIEW

File: newsletter-february-2022.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+and share what we'll be focusing on in the near future.
+
+Company updates
+---------------
+
+- We have mostly finished migrating Read the Docs for Business users to Cloudflare for SSL.
+  There are lots of interesting features this will enable,
+  so stay tuned for updates there.
+- We're sad to announce that Juan Luis has moved on from Read the Docs as our developer advocate.
+  The work he did was vital towards getting our :doc:`CZI grant </czi-grant-announcement>` mostly finished, and we thank him for his time spent bettering the RTD, Sphinx, and docs community.
```
Comment (astrojuanlu) [inline newsletter-february-2022.rst:27]: ❤️

File: newsletter-february-2022.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- We are working to define a policy for canonical docs and cloned versions,
+  so that we can more easily remove outdated docs for projects.
+- We're working to investigate supporting a CDN on Read the Docs for Business,
+  which will be an exciting new feature for our users there.
+- We're `looking at how to support <https://github.com/readthedocs/readthedocs.org/issues/8811>`_ multiple ``readthedocs.yml`` files in a single git repo.
+
+Possible issues
+---------------
+
+We continue to see a good place of development on the Sphinx project,
```
Comment (astrojuanlu) [inline newsletter-february-2022.rst:64]:
```suggestion
We continue to see a good pace of development on the Sphinx project,
```

?

File: newsletter-february-2022.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+- We have mostly finished migrating Read the Docs for Business users to Cloudflare for SSL.
+  There are lots of interesting features this will enable,
+  so stay tuned for updates there.
+- We're sad to announce that Juan Luis has moved on from Read the Docs as our developer advocate.
+  The work he did was vital towards getting our :doc:`CZI grant </czi-grant-announcement>` mostly finished, and we thank him for his time spent bettering the RTD, Sphinx, and docs community.
+- On a related note, we're going to be hiring again soon to fill another position.
+  It will be a bit different and likely a product-focused Python development position.
+  If you're interested, please `let us know`_.
+
+.. _let us know: mailto:ads@readthedocs.org?subject=Job%20Posting
```
Comment (agjohnson) [inline newsletter-february-2022.rst:32]: Guessing we don't want `ads` here.

```suggestion
.. _let us know: mailto:hello@readthedocs.org?subject=Job%20Posting
```

File: newsletter-february-2022.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+New features
+------------
+
+In January we continued to work on refactors and internal changed.
+Among the major user-facing changes:
+
+- We fixed a bug in BitBucket that didn't allow us to properly sync user permissions.
```
Comment (agjohnson) [inline newsletter-february-2022.rst:40]:
```suggestion
- We fixed a bug in Bitbucket that didn't allow us to properly sync user permissions.
```

**Pull request merged**